src/internal/cache-client.ts

```diff
@@ -1956,6 +1958,130 @@ export class CacheClient {
     });
   }
 
+  public async sortedSetRemoveElement(
+    cacheName: string,
+    sortedSetName: string,
+    value: string | Uint8Array
+  ): Promise<CacheSortedSetRemoveElement.Response> {
+    try {
+      validateCacheName(cacheName);
+      validateSortedSetName(sortedSetName);
+    } catch (err) {
+      return new CacheSortedSetFetch.Error(normalizeSdkError(err as Error));
+    }
+
+    this.logger.trace("Issuing 'sortedSetRemoveElement' request");
+
+    const result = await this.sendSortedSetRemoveElement(
+      cacheName,
+      this.convert(sortedSetName),
+      this.convert(value)
+    );
+
+    this.logger.trace(
+      "'sortedSetRemoveElement' request result: %s",
+      truncateString(result.toString())
+    );
+    return result;
+  }
+
+  private async sendSortedSetRemoveElement(
+    cacheName: string,
+    sortedSetName: Uint8Array,
+    value: Uint8Array
+  ): Promise<CacheSortedSetRemoveElement.Response> {
+    const request = new grpcCache._SortedSetRemoveRequest({
+      set_name: sortedSetName,
+      some: new grpcCache._SortedSetRemoveRequest._Some({
+        values: [value],
+      }),
+    });
+
+    const metadata = this.createMetadata(cacheName);
+    return await new Promise(resolve => {
+      this.clientWrapper.getClient().SortedSetRemove(
+        request,
+        metadata,
+        {
+          interceptors: this.interceptors,
+        },
+        err => {
+          if (err) {
+            resolve(
+              new CacheSortedSetRemoveElement.Error(
+                cacheServiceErrorMapper(err)
+              )
+            );
+          } else {
+            resolve(new CacheSortedSetRemoveElement.Success());
+          }
+        }
+      );
+    });
+  }
+
+  public async sortedSetRemoveElements(
+    cacheName: string,
+    sortedSetName: string,
+    values: string[] | Uint8Array[]
```

Comment:
We should validate these are not null and non-zero in length, right?

Reply:
i was planning on experimenting with edge cases via TDD once we have a pattern for testing, but didn't want to block making this available for early integrations since we aren't releasing it just yet.